---
course: data-engineering-zoomcamp
id: ba210ca8ef
question: I changed location in dbt, but dbt run still gives me an error
section: 'Module 4: analytics engineering with dbt'
sort_order: 2710
---

Remove the dataset from BigQuery which was created by dbt and run dbt run again so that it will recreate the dataset in BigQuery with the correct location

